cognite/client/_api/assets.py

```diff
@@ -1311,6 +1314,14 @@ def _make_asset_updates(self, asset: Asset, patch: bool) -> AssetUpdate:
         upd._update_object = dct_update
         return upd
 
+    def _get_latest_exception(self) -> Exception | None:
```

Comment:
This function is not used. Is it necessary to have?

Reply:
Oh that’s awkward 😂 I’ll remove it!